R/rmarkdown.R

```diff
@@ -51,12 +51,14 @@ render_rmarkdown <- function(pkg, input, output, ..., seed = NULL, copy_images =
       )
     ),
     error = function(cnd) {
+      # browser()
       cli::cli_abort(
         c(
           "Failed to render RMarkdown document.",
           x = gsub("\r", "", cnd$stderr, fixed = TRUE)
         ),
-        parent = cnd
+        parent = cnd$parent,
```

Comment:
@gaborcsardi is this safe to do? I only want to show the traceback from callr, not the traceback leading up to callr.

Reply:
With the caveat that if the error happens in the main process then you'll not see any of that. So maybe 
```
parent = cnd$parent %||% cnd
```
?